include/onnxruntime/core/providers/xnnpack/xnnpack_provider_factory.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#include "onnxruntime_c_api.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+ORT_EXPORT ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_Xnnpack,
```

Comment:
This is the old way of adding the EP that is deprecated. Instead of an opaque exported symbol the EP should be enabled via SessionOptionsAppendExecutionProvider

See https://github.com/microsoft/onnxruntime/pull/11798.

Reply:
Excluded 'xnnpack' for 'gen_def.py'